plugins/community/neoeinstein-prost/v0.3.1/buf.plugin.yaml

```diff
@@ -7,3 +7,24 @@ output_languages:
   - rust
 spdx_license_id: Apache-2.0
 license_url: https://github.com/neoeinstein/protoc-gen-prost/blob/protoc-gen-prost-v0.3.1/LICENSE
+registry:
+  cargo:
+    # https://github.com/neoeinstein/protoc-gen-prost/blob/protoc-gen-prost-v0.3.1/protoc-gen-prost/Cargo.toml#L21
+    # https://github.com/tokio-rs/prost/blob/v0.12.3/prost-build/Cargo.toml#L15
+    # TODO: Should we actually set this? This is just the rust-version set by
+    # prost-build, which isn't necessarily the rust-version that the *generated
+    # code* requires.
+    rust_version: "1.60"
```

Comment:
Left a couple of comments in here for the sake of discussion in this PR that I'm going to rip out.

To start off with, for both of these plugins, it's generally pretty difficult to understand what the generated code needs, both as a MSRV and dependencies.

For the MSRV, I think we should leave it unset for both plugins, for now. Determining the MSRV of a regular Rust crate is already tricky. [Here's the section describing the field](https://doc.rust-lang.org/cargo/reference/manifest.html#the-rust-version-field) from the reference, which links to [this somewhat scary-looking suggestion of using third-party tools to verify that the specified `rust-version` is _actually_ the MSRV for the crate](https://doc.rust-lang.org/cargo/guide/continuous-integration.html#verifying-rust-version). Similar to Python's `python-version`, unless the plugin or underlying generator specifically documents and tests the MSRV for generated code, I doubt we should populate this value.

Reply:
Agreed, even more so, if it's optional we should just exclude it unless there's a good reason (like it breaks a build) to include it.